alfred-telemetry-platform/src/main/java/eu/xenit/alfred/telemetry/binder/NodeMetrics.java

```diff
@@ -0,0 +1,81 @@
+package eu.xenit.alfred.telemetry.binder;
+
+import io.micrometer.core.instrument.Gauge;
+import io.micrometer.core.instrument.MeterRegistry;
+
+import java.util.concurrent.TimeUnit;
+import java.util.function.ToDoubleFunction;
+
+import io.micrometer.core.instrument.TimeGauge;
+import org.alfresco.repo.domain.node.NodeDAO;
+import org.alfresco.repo.domain.permissions.AclDAO;
+import org.alfresco.repo.transaction.RetryingTransactionHelper;
+import org.alfresco.service.transaction.TransactionService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class NodeMetrics {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(NodeMetrics.class);
+
+    public static final String ALC_PREFIX = "alfresco.alc";
+    public static final String DAO_PREFIX = "alfresco.node";
+    private TransactionService transactionService;
+    private TimeUnit baseTimeUnit = TimeUnit.MILLISECONDS;
+
+    private NodeDAO nodeDAO;
+    private AclDAO aclDAO;
+    private MeterRegistry registry;
+
+    public NodeMetrics(NodeDAO nodeDAOComponent, AclDAO aclDAOComponent, TransactionService transactionService,
+                       MeterRegistry registry, boolean enabled) {
+        this.nodeDAO = nodeDAOComponent;
+        this.aclDAO = aclDAOComponent;
+        this.transactionService = transactionService;
+        this.registry = registry;
+        if (enabled) {
+            registerMetrics();
+        }
+    }
+
+    private void registerMetrics() {
+        LOGGER.info("Registering Solr metrics");
+
+        Gauge.builder(String.format("%s.%s", DAO_PREFIX, "maxTxNode"), nodeDAO, wrapDAOFunction(NodeDAO::getMaxNodeId))
```

Comment:
```suggestion
        Gauge.builder(String.format("%s.%s", DAO_PREFIX, "maxNodeId"), nodeDAO, wrapDAOFunction(NodeDAO::getMaxNodeId))
```

Has nothing to do with transactions.